python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -163,21 +166,34 @@ cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
     cdef cudf_io_types.source_info source = make_source_info(
         filepaths_or_buffers)
 
-    # Setup parquet reader arguments
-    cdef read_parquet_args args = read_parquet_args(source)
+    cdef vector[string] cpp_columns
+    cdef bool cpp_strings_to_categorical = strings_to_categorical
+    cdef bool cpp_use_pandas_metadata = use_pandas_metadata
+    cdef size_type cpp_skip_rows = skip_rows if skip_rows is not None else 0
+    cdef size_type cpp_num_rows = num_rows if num_rows is not None else -1
+    cdef vector[vector[size_type]] cpp_row_groups
+    cdef data_type cpp_timestamp_type = cudf_types.data_type(
+        cudf_types.type_id.EMPTY
+    )
 
     if columns is not None:
-        args.columns.reserve(len(columns))
+        cpp_columns.reserve(len(columns))
         for col in columns or []:
-            args.columns.push_back(str(col).encode())
-    args.strings_to_categorical = strings_to_categorical
-    args.use_pandas_metadata = use_pandas_metadata
-
-    args.skip_rows = skip_rows if skip_rows is not None else 0
-    args.num_rows = num_rows if num_rows is not None else -1
+            cpp_columns.push_back(str(col).encode())
     if row_groups is not None:
-        args.row_groups = row_groups
-    args.timestamp_type = cudf_types.data_type(cudf_types.type_id.EMPTY)
+        cpp_row_groups = row_groups
+
+    cdef parquet_reader_options args
+    # Setup parquet reader arguments
+    args = move(parquet_reader_options.builder(source).
```

Comment:
I think `black` prefers the following:

```python
move(
     bar()
    .foo()
    .bar()
    .baz()
    ...
)
```

Unfortunately it doesn't support Cython yet so we need to do it manually